schemas/rum/_perf-info-schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "rum/_perf-info-schema.json",
+  "title": "RumPerfInfo",
+  "type": "object",
+  "description": "Schema of properties for a technical performance metic",
+  "required": ["view_min", "view_max", "view_average"],
+  "properties": {
+    "view_min": {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏maybe remove the `view_` prefix, as it is already in a `view` namespace I think it is already explicit enough.